VSRAD.Package/DebugVisualizer/VisualizerTable.cs

```diff
@@ -302,19 +308,29 @@ private static void LockWatchRowForEditing(DataGridViewRow row, bool canBeRemove
 
         #region Styling
 
-        public void GrayOutColumns(uint groupSize)
+        public void GrayOutColumns()
```

Comment:
The name should be changed too (`GrayOutRows`?)

Reply:
Thanks! Resolved in c1dce41